core/templates/css/oppia.css

```diff
@@ -4380,8 +4380,9 @@ oppia-noninteractive-link {
   border-radius: 4px;
   -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
   box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
-  height: auto;
+  max-height: 125px;
   min-height: 80px;
+  overflow-y: scroll;
```

Comment:
Instead of this, can we just have a word-break property to handle wrapping of large text? e.g. `word-break: break-word;`

Reply:
@kevintab95, but I guess it won't solve the problem of no side scroll for large texts. There will still be an overflow if the text input is too long.

![Screenshot from 2020-12-17 13-58-50](https://user-images.githubusercontent.com/66058206/102462765-3be04380-4070-11eb-9f80-65669a10d254.png)

Reply:
Not sure I follow.. In the example above, all the text is visible, right?

Reply:
@kevintab95, yes it is visible. But I think we need a side scroll for large input as if the user enters a large text, the height of the text box will keep on increasing and it will cause inconvenience for the user as they will have to scroll a lot to add a new option or reach a particular option to edit it.

Reply:
Note that the class you are modifying is not restricted to multiple choice input interaction, it is used by all RTE components so we should not add a fix that would add a fixed height in all places. Let us just add the word break for fixing the overflow issue for now.

For large inputs in the interaction options, I am not sure if the fix should be to add scrolling. Maybe we should prevent or advise users against having large inputs inside options in a multiple choice input interaction -- like you mentioned, this can be a bit annoying for learners viewing this interaction. But this is a different issue in itself, perhaps file another issue for it so it can be tracked. Does this make sense?

Reply:
Yeah, that seems like a better idea. I'll updated this PR accordingly. Thanks! :)